_posts/2001-01-03-customize-project.md

```diff
@@ -130,6 +130,27 @@ The access restriction feature is intended for use with special roles that can b
 
 ------------
 
+## Webhooks
+
+<figure>
+<img alt="Project Webhooks Settings tab" src="{{ site.url }}/images/341-project-webhooks-settings.png" />
+<figcaption>Project Webhooks Settings tab</figcaption>
+</figure>
+
+Webhooks feature are HTTP callbacks which triggered when a document in a language has reached a certain milestone. Currently, webhooks event will be trigger when
+
+- A document has reached 100% Translated
+- A document has reached 100% Approved (By reviewer)
+
+When event occurs, Zanata will make an HTTP POST to the URI configured in the project.
```

Comment:
When **an** event occurs, Zanata will make a~~n~~ HTTP POST to the URI configured in the project.